### PR TITLE
Front end redesign

### PR DESCRIPTION
**Pull request by JeanVegaD**
# What does this PR do?

#202 
- apply initial front-end redesign 

# How this should be tested?
- go to http://localhost:3000

### CODE REVIEW

File: webapp/src/routes/Home/ShowSponsors.js [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+    const num = 55
+
+    if (str.length <= num) {
+      return str
+    }
+
+    return str.slice(0, num) + '...'
+  }
+
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowSponsors.js:254]: Remove the whitespace here

File: webapp/src/routes/Home/ShowSponsors.js [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+      </Card>
+    )
+  }
+
+  SponsorCard.propTypes = {
+    name: PropTypes.string,
+    description: PropTypes.string,
+  }
+
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowSponsors.js:285]: Please remove this whitespace

File: webapp/src/routes/Home/ShowSponsors.js [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+          </Grid>
+        ))}
+      </>
+    )
+  }
+
+  const truncateString = (str) => {
+    const num = 55
+
+    if (str.length <= num) {
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowSponsors.js:247]: If the conditional has only one line in its body, you can remove the brackets

File: webapp/src/routes/Home/ShowSponsors.js [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+    )
+  }
+
+  SponsorItem.propTypes = {
+    name: PropTypes.string,
+    bussines_type: PropTypes.string,
+  }
+
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowSponsors.js:204]: Please remote this white space

File: webapp/src/routes/Home/ShowSponsors.js [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import Avatar from '@material-ui/core/Avatar';
+import StorefrontIcon from '@material-ui/icons/Storefront';
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowSponsors.js:19]: Please remove this whitespace

File: webapp/src/routes/Home/ShowOffers.js [hunk 6]
```diff
@@ -0,0 +1,345 @@
...
+      }
+    </>
+
+
+
+  )
+}
+
+
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowOffers.js:336]: Please remove this whitespaces

File: webapp/src/routes/Home/ShowOffers.js [hunk 7]
```diff
@@ -0,0 +1,345 @@
...
+          alignItems="center"
+          className={classes.offersGridContainer}
+          spacing={2}
+        >
+          <LoadOfferDesktop />
+        </Grid>
+      }
+    </>
+
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowOffers.js:330]: Remove this whitespaces

File: webapp/src/routes/Home/ShowOffers.js [hunk 8]
```diff
@@ -0,0 +1,345 @@
...
+            img={offer.images}
+          />
+        ))}
+      </>
+    )
+  }
+
+  const OfferItem = (props) => {
+
+    return (
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowOffers.js:202]: please remove this white space

Comment (JustinCast) [reply webapp/src/routes/Home/ShowOffers.js:202]: When the components only has a return in its JSX content, you can do something like
```
const OfferItem = (props) => (
    JSX HERE
)
```

File: webapp/src/routes/Home/ShowOffers.js [hunk 9]
```diff
@@ -0,0 +1,345 @@
...
+    textAlign: "center",
+    color: "#121212",
+  }
+}))
+
+
+const ShowOffers = ({ offers, loading, isDesktop }) => {
+  const classes = useStyles()
+
+  const LoadOffers = () => {
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowOffers.js:170]: When the components only has a return in its JSX content, you can do something like

```
const OfferItem = (props) => (
    JSX HERE
)
```

File: webapp/src/routes/Home/ShowOffers.js [hunk 10]
```diff
@@ -0,0 +1,345 @@
...
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.14",
+    letterSpacing: "1",
+    textAlign: "center",
+    color: "#121212",
+  }
+}))
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowOffers.js:165]: Please remove this whitespace

File: webapp/src/routes/Home/ShowLifebanks.js [hunk 11]
```diff
@@ -0,0 +1,316 @@
...
+      </Card>
+    )
+  }
+
+  BankCard.propTypes = {
+    name: PropTypes.string,
+    description: PropTypes.string,
+  }
+
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowLifebanks.js:284]: Please remove this whitespace

File: webapp/src/routes/Home/ShowLifebanks.js [hunk 12]
```diff
@@ -0,0 +1,316 @@
...
+    if (str.length <= num) {
+      return str
+    }
+
+    return str.slice(0, num) + '...'
+  }
+
+
+  const BankCard = (props) => {
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowLifebanks.js:254]: When the components only has a return in its JSX content, you can do something like

```
const OfferItem = (props) => (
    JSX HERE
)
```

File: webapp/src/routes/Home/ShowLifebanks.js [hunk 13]
```diff
@@ -0,0 +1,316 @@
...
+  const truncateString = (str) => {
+    const num = 55
+
+    if (str.length <= num) {
+      return str
+    }
+
+    return str.slice(0, num) + '...'
+  }
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowLifebanks.js:252]: Please remove this whitespace

File: webapp/src/routes/Home/ShowLifebanks.js [hunk 14]
```diff
@@ -0,0 +1,316 @@
...
+          </Grid>
+        ))}
+      </>
+    )
+  }
+
+  const truncateString = (str) => {
+    const num = 55
+
+    if (str.length <= num) {
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowLifebanks.js:246]: Unnecessary curly braces here

File: webapp/src/routes/Home/ShowLifebanks.js [hunk 15]
```diff
@@ -0,0 +1,316 @@
...
+
+    )
+  }
+
+  BankItem.propTypes = {
+    name: PropTypes.string,
+    description: PropTypes.string,
+  }
+
+  const LoadBanksDesktop = () => {
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowLifebanks.js:204]: When the components only has a return in its JSX content, you can do something like

```
const OfferItem = (props) => (
    JSX HERE
)
```

File: webapp/src/routes/Home/ShowLifebanks.js [hunk 16]
```diff
@@ -0,0 +1,316 @@
...
+        <ListItemText
+          primary={
+            <Typography className={classes.listItemPrimaryText} noWrap variant="body2">{props.name}</Typography>
+          }
+          secondary={
+            <Typography className={classes.listItemSecondaryText} noWrap variant="body2">{props.description}</Typography>
+          }
+        />
+      </ListItem>
+
```
Comment (JustinCast) [inline webapp/src/routes/Home/ShowLifebanks.js:195]: Please remove this whitespace

File: webapp/src/components/Notification/Notification.js [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+          </Toolbar>
+        </AppBar>
+        <List />
+      </Dialog>
+    </>
+  )
+}
+
+Notification.propTypes = {
+
```
Comment (JustinCast) [inline webapp/src/components/Notification/Notification.js:99]: Please remove this whitespace here

**Review by JustinCast — CHANGES_REQUESTED** [review]

**Review by JustinCast — APPROVED** [review]

**Review by AngeloCG97 — APPROVED** [review]

**Pull request merged**